docs/DeveloperGuide.md

```diff
@@ -120,16 +120,17 @@ The sections below give more details of each component.
 ![Structure of the UI Component](images/UiClassDiagram.png)
 
 **API** :
-[`Ui.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/ui/Ui.java)
+[`Ui.java`](https://github.com/AY2021S2-CS2103-T14-1/tp/blob/master/src/main/java/seedu/address/ui/Ui.java)
 
-The UI consists of a `MainWindow` that is made up of parts e.g.`CommandBox`, `ResultDisplay`, `PersonListPanel`, `StatusBarFooter` etc. All these, including the `MainWindow`, inherit from the abstract `UiPart` class.
+The UI consists of a `MainWindow` that is made up of parts e.g.`CommandBox`, `ResultDisplay`, `ListPanel`s, `StatusBarFooter` etc. All these, including the `MainWindow`, inherit from the abstract `UiPart` class. `XYZListPanel` is a placeholder for a concrete ListPanel. Currently `Resident`, `Room`, and `Issue` each have a ListPanel class. Similarly for `XYZCard`.
```

Comment:
Should the last "ListPanel" be `ListPanel`?

Reply:
Also, maybe shift the `XYZListPanel` and `XYZCard` part to a "**Diagram Notes** :" section